### ISSUE TITLE
Weird Issue 

### ISSUE THREAD

**Issue by Adblu** [pandas-profiling/pandas-profiling#165]
When I try to run profiling, I got an error:

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\spawn.py", line 105, in spawn_main
Traceback (most recent call last):
  File "<string>", line 1, in <module>
    exitcode = _main(fd)
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\spawn.py", line 114, in _main
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\spawn.py", line 105, in spawn_main
    prepare(preparation_data)
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\spawn.py", line 225, in prepare
    exitcode = _main(fd)
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\spawn.py", line 114, in _main
    _fixup_main_from_path(data['init_main_from_path'])
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\spawn.py", line 277, in _fixup_main_from_path
    prepare(preparation_data)
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\spawn.py", line 225, in prepare
    run_name="__mp_main__")
  File "C:\Users\N\Anaconda3\envs\ut\lib\runpy.py", line 263, in run_path
    _fixup_main_from_path(data['init_main_from_path'])
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\spawn.py", line 277, in _fixup_main_from_path
    run_name="__mp_main__")
  File "C:\Users\N\Anaconda3\envs\ut\lib\runpy.py", line 263, in run_path
    pkg_name=pkg_name, script_name=fname)
  File "C:\Users\N\Anaconda3\envs\ut\lib\runpy.py", line 96, in _run_module_code
    pkg_name=pkg_name, script_name=fname)
  File "C:\Users\N\Anaconda3\envs\ut\lib\runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "C:\Users\N\Anaconda3\envs\ut\lib\runpy.py", line 85, in _run_code
    mod_name, mod_spec, pkg_name, script_name)
  File "C:\Users\N\Anaconda3\envs\ut\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "D:\Praca\Python\ut\clean_hikes_file.py", line 8, in <module>
    profile = pandas_profiling.ProfileReport(data)
  File "C:\Users\N\Anaconda3\envs\ut\lib\site-packages\pandas_profiling\__init__.py", line 66, in __init__
    exec(code, run_globals)
  File "D:\Praca\Python\ut\clean_hikes_file.py", line 8, in <module>
    profile = pandas_profiling.ProfileReport(data)
  File "C:\Users\N\Anaconda3\envs\ut\lib\site-packages\pandas_profiling\__init__.py", line 66, in __init__
    description_set = describe_df(df, **kwargs)
  File "C:\Users\N\Anaconda3\envs\ut\lib\site-packages\pandas_profiling\describe.py", line 361, in describe
    description_set = describe_df(df, **kwargs)
  File "C:\Users\N\Anaconda3\envs\ut\lib\site-packages\pandas_profiling\describe.py", line 361, in describe
    pool = multiprocessing.Pool(pool_size)
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\context.py", line 119, in Pool
    pool = multiprocessing.Pool(pool_size)
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\context.py", line 119, in Pool
    context=self.get_context())
    context=self.get_context())
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\pool.py", line 174, in __init__
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\pool.py", line 174, in __init__
    self._repopulate_pool()
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\pool.py", line 239, in _repopulate_pool
    self._repopulate_pool()
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\pool.py", line 239, in _repopulate_pool
    w.start()
    w.start()
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\process.py", line 105, in start
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\process.py", line 105, in start
    self._popen = self._Popen(self)
    self._popen = self._Popen(self)
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\context.py", line 322, in _Popen
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\context.py", line 322, in _Popen
    return Popen(process_obj)
    return Popen(process_obj)
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\popen_spawn_win32.py", line 33, in __init__
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\popen_spawn_win32.py", line 33, in __init__
    prep_data = spawn.get_preparation_data(process_obj._name)
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\spawn.py", line 143, in get_preparation_data
    prep_data = spawn.get_preparation_data(process_obj._name)
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\spawn.py", line 143, in get_preparation_data
    _check_not_importing_main()
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\spawn.py", line 136, in _check_not_importing_main
    _check_not_importing_main()
  File "C:\Users\N\Anaconda3\envs\ut\lib\multiprocessing\spawn.py", line 136, in _check_not_importing_main
    is not going to be frozen to produce an executable.''')
RuntimeError: 
        An attempt has been made to start a new process before the
        current process has finished its bootstrapping phase.
        This probably means that you are not using fork to start your
        child processes and you have forgotten to use the proper idiom
        in the main module:
            if __name__ == '__main__':
                freeze_support()
                ...
        The "freeze_support()" line can be omitted if the program
        is not going to be frozen to produce an executable.
    is not going to be frozen to produce an executable.''')
RuntimeError: 
        An attempt has been made to start a new process before the
        current process has finished its bootstrapping phase.
        This probably means that you are not using fork to start your
        child processes and you have forgotten to use the proper idiom
        in the main module:
            if __name__ == '__main__':
                freeze_support()
                ...
        The "freeze_support()" line can be omitted if the program
        is not going to be frozen to produce an executable.
```
And it goes on and on an on.

Any guess whats happening ?

**Comment by Adblu** [comment]
After installing pandas profiler through conda, error changed to:
```
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "C:\Program Files\JetBrains\PyCharm 2018.3\helpers\pydev\_pydev_bundle\pydev_umd.py", line 198, in runfile
    pydev_imports.execfile(filename, global_vars, local_vars)  # execute the script
  File "C:\Program Files\JetBrains\PyCharm 2018.3\helpers\pydev\_pydev_imps\_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "D:/Praca/Python/ut/clean_hikes_file.py", line 3, in <module>
    import pandas_profiling
  File "C:\Program Files\JetBrains\PyCharm 2018.3\helpers\pydev\_pydev_bundle\pydev_import_hook.py", line 21, in do_import
    module = self._system_import(name, *args, **kwargs)
  File "C:\Users\N\Anaconda3\envs\ut\lib\site-packages\pandas_profiling\__init__.py", line 11, in <module>
    from .describe import describe
  File "C:\Program Files\JetBrains\PyCharm 2018.3\helpers\pydev\_pydev_bundle\pydev_import_hook.py", line 21, in do_import
    module = self._system_import(name, *args, **kwargs)
  File "C:\Users\N\Anaconda3\envs\ut\lib\site-packages\pandas_profiling\describe.py", line 13, in <module>
    from pandas_profiling.plot import histogram, mini_histogram
  File "C:\Program Files\JetBrains\PyCharm 2018.3\helpers\pydev\_pydev_bundle\pydev_import_hook.py", line 21, in do_import
    module = self._system_import(name, *args, **kwargs)
  File "C:\Users\N\Anaconda3\envs\ut\lib\site-packages\pandas_profiling\plot.py", line 15, in <module>
    matplotlib.use(BACKEND)
  File "C:\Program Files\JetBrains\PyCharm 2018.3\helpers\pydev\pydev_ipython\matplotlibtools.py", line 70, in patched_use
    matplotlib.real_use(*args, **kwargs)
  File "C:\Program Files\JetBrains\PyCharm 2018.3\helpers\pydev\pydev_ipython\matplotlibtools.py", line 70, in patched_use
    matplotlib.real_use(*args, **kwargs)
  File "C:\Program Files\JetBrains\PyCharm 2018.3\helpers\pydev\pydev_ipython\matplotlibtools.py", line 70, in patched_use
    matplotlib.real_use(*args, **kwargs)
  [Previous line repeated 476 more times]
RecursionError: maximum recursion depth exceeded
```

**Comment by sbrugman** [comment]
Same issue as #134, you can find the solution there.

**Comment by Bibaceta** [comment]
sbrugman you closed #134 without replying something very important: where to add  if __name__=="__main__":
